.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "test/bats"]
+	path = test/bats
+	url = https://github.com/bats-core/bats-core.git
+[submodule "test/test_helper/bats-support"]
+	path = test/test_helper/bats-support
+	url = https://github.com/bats-core/bats-support.git
+[submodule "test/test_helper/bats-assert"]
+	path = test/test_helper/bats-assert
+	url = https://github.com/bats-core/bats-assert.git
```

Comment:
submodules can be painful to manage over time. We use them of course, I'm just wondering if we should start using fewer of them, especially when we're installing other folks' software. Should we instead have a makefile that downloads exact releases and builds them using their build instructions?